api/schema_responses/serializers.py

```diff
@@ -0,0 +1,108 @@
+from api.base.serializers import (
+    JSONAPISerializer,
+    LinksField,
+    RelationshipField,
+)
+
+from rest_framework import serializers as ser
+from api.base.utils import absolute_reverse
+from osf.models.schema_responses import SchemaResponses
+from osf.models import Registration
+from rest_framework import exceptions
+from django.utils import timezone
+
+
+class SchemaResponsesSerializer(JSONAPISerializer):
+    id = ser.CharField(required=False, source='_id', read_only=True)
+    title = ser.CharField(required=False, allow_blank=True)
```

Comment:
Do we need a title here? Doesn't the title live on the registration itself?

Reply:
Good point, no reason to have a title I can see.